package.json

```diff
@@ -13,78 +13,60 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.11.0",
-    "@bcrs-shared-components/bread-crumb": "1.0.2",
-    "@bcrs-shared-components/business-lookup": "1.0.2",
-    "@bcrs-shared-components/certify": "2.0.0",
-    "@bcrs-shared-components/completing-party": "2.0.6",
-    "@bcrs-shared-components/confirm-dialog": "1.1.0",
-    "@bcrs-shared-components/contact-info": "1.1.2",
-    "@bcrs-shared-components/corp-type-module": "1.0.6",
-    "@bcrs-shared-components/court-order-poa": "2.0.0",
-    "@bcrs-shared-components/date-picker": "1.1.15",
-    "@bcrs-shared-components/document-delivery": "1.0.0",
-    "@bcrs-shared-components/effective-date-time": "1.0.2",
-    "@bcrs-shared-components/enums": "1.0.19",
-    "@bcrs-shared-components/help-business-number": "1.0.0",
-    "@bcrs-shared-components/interfaces": "1.0.39",
-    "@bcrs-shared-components/mixins": "1.0.6",
-    "@bcrs-shared-components/nature-of-business": "1.0.3",
-    "@bcrs-shared-components/staff-comments": "1.2.0",
-    "@bcrs-shared-components/staff-payment": "2.0.1",
-    "@bcrs-shared-components/web-chat": "2.0.0",
-    "@mdi/font": "^5.5.55",
```

Comment:
I moved this to Dev Dependencies as per some examples I saw. It will be pretty obvious if this change doesn't work (once the project builds and runs).